### ISSUE TITLE
User-guide section on hyper-parameter optimization

### ISSUE THREAD

**Issue by hrzn** [unit8co/darts#1125]

**Comment by TamerAbdelmigid** [comment]
It would be super helpful if you could include an example on using PopulationBasedTraining of ray tune to tune a learning rate schedule of DARTS Torch models.
